daemon/api.go

```diff
@@ -186,6 +187,12 @@ var (
 		POST:   postBuy,
 	}
 
+	readyToBuyCmd = &Command{
+		Path:   "/v2/buy/ready",
```

Comment:
Why do we need this as a method in the API? I'd expect that to be done implicitly every time someone attempts to buy instead.